### PR TITLE
feat: add bluemap support

### PR DESCRIPTION
**Pull request by Jarva**
Adds support for [Bluemap](https://github.com/BlueMap-Minecraft/BlueMap) on Forge and Fabric. 

Bluemap does not persist markers itself and requires them all to be added when the API is loaded.

Bluemap also separates MarkerSets by world, which requires us to have access to MinecraftServer before registering the claim markers.

### CODE REVIEW

File: fabric/src/main/java/io/github/flemmli97/flan/fabric/FlanFabric.java [hunk 1]
```diff
@@ -73,6 +74,8 @@ public void onInitialize() {
     public static void serverLoad(MinecraftServer server) {
         Flan.lockRegistry(server);
         ConfigHandler.serverLoad(server);
+        if (FabricLoader.getInstance().isModLoaded("bluemap"))
```
Comment (Flemmli97) [inline fabric/src/main/java/io/github/flemmli97/flan/fabric/FlanFabric.java:77]: Might be better to move this into the mod constructor like dynmap. Is there a reason why its in this event?

File: forge/src/main/java/io/github/flemmli97/flan/forge/forgeevent/ServerEvents.java [hunk 2]
```diff
@@ -12,12 +13,16 @@
...
 import net.minecraftforge.event.server.ServerStartedEvent;
+import net.minecraftforge.fml.ModList;
 
 public class ServerEvents {
 
     public static void serverStart(ServerAboutToStartEvent event) {
         Flan.lockRegistry(event.getServer());
         ConfigHandler.serverLoad(event.getServer());
+
+        if (ModList.get().isLoaded("bluemap"))
```
Comment (Flemmli97) [inline forge/src/main/java/io/github/flemmli97/flan/forge/forgeevent/ServerEvents.java:24]: Might be better to move this into the mod constructor like dynmap. Is there a reason why its in this event?

**Comment by Jarva** [comment]
We require access to MinecraftServer to process all available worlds, which requires us to use the server start event instead of directly using the mod constructor

**Comment by Flemmli97** [comment]
apparently i cant read. my bad. ye nothing wrong with it

**Pull request merged**